WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Categories/ProductCategoryListViewController.swift

```diff
@@ -115,7 +115,8 @@ private extension ProductCategoryListViewController {
         let options = GhostOptions(displaysSectionHeader: false,
                                    reuseIdentifier: ProductCategoryTableViewCell.reuseIdentifier,
                                    rowsPerSection: placeholderCategoriesPerSection)
-        ghostTableView.displayGhostContent(options: options)
+        ghostTableView.displayGhostContent(options: options,
+                                           style: .wooDefaultGhostStyle)
```

Comment:
nice I didn't know this is the fix for the ghost style! I will apply the same fix for https://github.com/woocommerce/woocommerce-ios/issues/1813 😄